@trevormccasland, it appears to be a unit test that is failing:

```
test_get_user_recommendations (tests.test_reddit.TestReddit) ... ok
Traceback (most recent call last):
  File "/home/travis/build/drewwalters96/subber/subber/config.py", line 43, in get_config
    with open(config_file) as f:
FileNotFoundError: [Errno 2] No such file or directory: 'subber.cfg'
```

--- a/tests/test_subber.py
+++ b/tests/test_subber.py
@@ -19,8 +19,9 @@ import flask_testing
 
 class TestSubber(flask_testing.TestCase):
 
+    @patch('subber.config.get_config')
     @patch('subber.reddit.Reddit')
-    def create_app(self, mock_session):
+    def create_app(self, mock_session, mock_config):
         # Mock session return value
         redditor = {'name': 'u/test',
                     'created': 2545896143}

